demo/index.html

```diff
@@ -1,59 +1,31 @@
 <!doctype html>
 <html>
-
```

Comment:
There'll be lots of diffs in the demo files due to the old Polymer 3 conversion process, but I've tried to keep them mostly the same.